### ISSUE TITLE
Documentation of --reduce-only glob syntax

### ISSUE THREAD

**Issue by JohanEngelen** [CyberShadow/DustMite#72]
I am not a glob guru, and it is not easy to check whether indeed dustmite is only reducing the files one wants to reduce: it would greatly help me if there'd be some examples of how to use `--reduce-only`.
In my current case, I want to only reduce files in the `fs` directory, which is inside a subdirectory, and also contains subdirectories itself. I'm guessing `--reduce-only="**/fs/*"` will do the trick, but I really don't know.

Thanks :)

**Comment by CyberShadow** [comment]
Yep, documentation in DustMite was lacking. It's addressed in the successor `reduce` (see the section "SELECTION" in the [man page](https://cy.md/reduce/)), but the implementation doesn't match the documentation yet there :)

I believe your guess is correct, except the double asterisk is redundant - only one is needed (i.e. `*/fs/*`). The matching function is [`globMatch`](https://dlang.org/library/std/path/glob_match.html).
